### ISSUE TITLE
Talkback / Voiceover (iOS & macOS) - Does not read out timestamp 

### ISSUE THREAD

**Issue by gavinspence** [bbc/simorgh#5519]
Talkback / Voiceover (mac OS) - Does not read out timestamp 

Timestamp is not read out when using TalkBack - in the swipe order, it is skipped.

Could be a duplicate of #5515

**Comment by gavinspence** [comment]
This is a screenreader issue which we will not be addressing.
